nomad/structs/structs.go

```diff
@@ -831,6 +832,11 @@ type ServerMember struct {
 	DelegateCur uint8
 }
 
+// ClusterMetadata is used to store per-cluster metadata.
+type ClusterMetadata struct {
+	ClusterID string
```

Comment:
A `CreateTime int64` field might be nice to mark when this cluster first established leadership? Seems extremely low value even for purely informational metadata so feel free to ignore.

Reply:
To that end, an UpdateTime might be a future nice to have as well.

Reply:
This should only ever be written once (until we add other fields), so if Update != Create you are having a very bad Time. :sweat_smile: